.ci/install_cc_shim.sh

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+set -e
+
+echo "Retrieve cc-shim repository"
+go get -d github.com/clearcontainers/shim || true
```

Comment:
We have to leave this as is for now because `go get` always return an exit code different from 0 for the shim. Even using the `-d` flag which is supposed to download only, we get the following error:
```
can't load package: package github.com/clearcontainers/shim: no buildable Go source files in /home/sebastien/gopath/src/github.com/clearcontainers/shim
```
That's the only way to make this test passing (unless you have a better proposal ;) ).